.travis.yml

```diff
@@ -3,41 +3,63 @@
 osx_image: xcode8.3
 sudo: required
 dist: trusty
-language: c
+
+language: node_js
+node_js:
+  - 8
+
 matrix:
   include:
   - os: osx
   - os: linux
     env: CC=clang CXX=clang++ npm_config_clang=1
     compiler: clang
+
 cache:
   directories:
   - node_modules
   - "$HOME/.electron"
   - "$HOME/.cache"
+
 addons:
   apt:
     packages:
     - libgnome-keyring-dev
     - icnsutils
+    - graphicsmagick
+    - xz-utils
     #- xvfb
+
 before_install:
-- mkdir -p /tmp/git-lfs && curl -L https://github.com/github/git-lfs/releases/download/v1.2.1/git-lfs-$([
-  "$TRAVIS_OS_NAME" == "linux" ] && echo "linux" || echo "darwin")-amd64-1.2.1.tar.gz
-  | tar -xz -C /tmp/git-lfs --strip-components 1 && /tmp/git-lfs/git-lfs pull
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install --no-install-recommends -y icnsutils graphicsmagick xz-utils; fi
+  - if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get -qq update ; fi
+  #- if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update ; fi
+
 install:
-#- export DISPLAY=':99.0'
-#- Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 &
-- nvm install 8
-- curl -o- -L https://yarnpkg.com/install.sh | bash
-- source ~/.bashrc
-- npm install -g xvfb-maybe
-- yarn
+  #- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then export DISPLAY=':99.0' ; fi
+  #- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then Xvfb :99 -screen 0 1024x768x24 > /dev/null 2>&1 & ; fi
```

Comment:
@Jocs Do you still need that?

Reply:
This file is generated by `electron-vue`,  I just use Travis CI to generate release draft.